testsuite/cluster-test/src/main.rs

```diff
@@ -342,30 +352,69 @@ impl BasicSwarmUtil {
 
 impl ClusterUtil {
     pub fn setup(args: &Args) -> Self {
-        let aws = Aws::new();
-        let cluster = Cluster::discover(&aws).expect("Failed to discover cluster");
-        let cluster = if args.peers.is_empty() {
-            cluster
-        } else {
-            cluster.validator_sub_cluster(args.peers.clone())
-        };
-        let prometheus = Prometheus::new(
-            cluster
-                .prometheus_ip()
-                .expect("Failed to discover prometheus ip in aws"),
-            aws.workspace(),
-        );
-        info!(
-            "Discovered {} validators and {} fns in {} workspace",
-            cluster.validator_instances().len(),
-            cluster.fullnode_instances().len(),
-            aws.workspace()
-        );
-        Self {
-            cluster,
-            aws,
-            prometheus,
-        }
+        Runtime::new().unwrap().block_on(async move {
```

Comment:
We should probably just move `Rutnime` from ClustreTestRunner to ClusterUtil and use it here.

Reply:
Right now I want to leave this as is. We will need a huge refactor once we migrate completely to kubernetes, where I imagine we will have a single global Runtime initiated in main using `#[tokio::main]`

Reply:
sounds good re refactor later

I don't think we need to migrate to tokio::main, we want to configure runtime ourselves